### PR TITLE
Immutable complex command (+ compound hierarchy cleanup)

### PR DESCRIPTION
**Pull request by cheretka**

### CODE REVIEW

File: src/test/java/CompoundCommandCopyTest.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import edu.kis.powp.jobs2d.command.CompoundCommand;
-import edu.kis.powp.jobs2d.command.DriverCommand;
-import edu.kis.powp.jobs2d.command.OperateToCommand;
-import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.driverCommands.ICompoundCommands.CompoundCommand;
```
Comment (kowallus) [inline src/test/java/CompoundCommandCopyTest.java]: I am not sure if creating new packages with such long names is necessary. Convention looks invalid (uppercase). Maybe resign from such refactoring.

**Review by kowallus — CHANGES_REQUESTED** [review]
To extensive package refactor. Please check if there are usages of other complex commands which could be replaced with the immutable implementation.
Next time, also resolve conflicts!

**Comment by cheretka** [comment]
- commands replaced with the immutable implementation 
- changed the hierarchy in a different way

File: src/test/java/CompoundCommandCopyTest.java (hunk 1 above)
Comment (cheretka) [reply src/test/java/CompoundCommandCopyTest.java]: done

**Pull request merged**